pandas/_libs/tslibs/period.pyx

```diff
@@ -1217,6 +1217,32 @@ cdef class _Period(object):
 
     @property
     def day(self):
+        """
+        Return the number of days of particular month of a given Period.
+
+        This attribute returns the total number of days of given month on which
+        the particular date occurs.
+        
+        Returns
+        -------
+        Int
+        Number of days till particular date
+
+        See also
+        --------
+        Period.dayofweek
+            Return the day of the week
```

Comment:
In this section the description should be on the same line as the referenced item separated by a " : "

Reply:
mean ===>>>>(Period.dayofweek: Return the day of the week)

Reply:
Put a space before the colon and change the word "Return" to "Get" and yes you are correct